flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/rules/logical/DecomposeGroupingSetsRule.scala

```diff
@@ -328,7 +328,7 @@ class DecomposeGroupingSetsRule
         val res: Long = call.getArgList.foldLeft(0L)(
           (res, arg) => (res << 1L) + (if (groups.contains(arg)) 0L else 1L))
         builder.makeLiteral(res, call.getType, false)
-      case _ => builder.constantNull()
```

Comment:
Deprecated in https://github.com/apache/calcite/commit/b432756e2be9ad0557a56254550eb4438dd0efcf
in favor of `#makeNullLiteral(RelDataType)`
because 

> because it produces untyped NULL literals that make planning difficult